README.md

```diff
@@ -16,14 +16,13 @@ The OSM project builds on the ideas and implementations of many cloud native eco
   - [Features](#features)
   - [SMI Specification Support](#smi-specification-support)
 - [OSM Design](#osm-design)
-- [Getting Started](#getting-started)
+- [Install](#install)
```

Comment:
`Getting Started` seems reasonable and easier to jump onto than `Install`. Would suggest to keep this as is, with a section on install.